nntoolkit/utils.py

```diff
@@ -92,7 +85,19 @@ def get_model(modelfile):
     tarfolder = tempfile.mkdtemp()
     tar.extractall(path=tarfolder)
     tar.close()
-    model_yml = yaml.load(open(os.path.join(tarfolder, 'model.yml')))
+    return True,tarfolder
```

Comment:
This is not good. It should only return "tarfolder".

You can return "None" for the error-case and check "[returnvalue] is None: ..."